tss-esapi/tests/integration_tests/context_tests/tpm_commands/non_volatile_storage_tests.rs

```diff
@@ -328,3 +328,90 @@ mod test_nv_read {
         assert_eq!(expected_data, actual_data);
     }
 }
+
+mod test_nv_increment {
+    use crate::common::create_ctx_with_session;
+    use std::convert::TryInto;
+    use tss_esapi::{
+        attributes::NvIndexAttributesBuilder,
+        constants::nv_index_type::NvIndexType,
+        handles::NvIndexTpmHandle,
+        interface_types::{
+            algorithm::HashingAlgorithm,
+            resource_handles::{NvAuth, Provision},
+        },
+        structures::NvPublicBuilder,
+    };
+    #[test]
```

Comment:
Newline before `#[test]`. Weird that `cargo fmt` doesn't catch this :thinking:

Reply:
Fixed